### ISSUE TITLE
Please release a new version to marketplace

### ISSUE THREAD

**Issue by brunetton** [nmsmith22389/vscode-incrementor#14]
Hi, I don't know if you're using VScode anymore, but please bump a new version to market place ! It would make everybody benefits from #5  and #6.

If you haven't time for that, let community know and add people to project !

Thanks

**Comment by duianto** [comment]
Seconded, the marketplace version needs to be updated.

Issue #5 (that seems to have been fixed in sept 25, 2017) almost made me uninstall the extension.
I installed it yesterday and tried to `C-Up` increment a number at the start of a line, but got the message:
>Running the contributed command:'incrementor.incByOne' failed.

That made me think that it might be an old extension, that doesn't work with the current vscode version.
But when I checked the issues and found #5 then I tried adding a space before the number and it worked as expected.

**Environment:**

- Extension Name: Incrementor
- Extension Version: 0.1.0
- OS Version: Windows 10 Version 1709 (OS Build 16299.371)
- VSCode version: 1.22.2

**Comment by nmsmith22389** [comment]
Yeah it is an old version and needs some TLC. I've just been so busy but I have some free time now and am going to compile all the PRs and release it as well as start working on it some more in the future.
